src/cookbook/initializing.md

```diff
@@ -71,6 +73,21 @@ location:
     $ echo 'PYTHONPATH=$PYTHONPATH:/usr/local/samba/lib/python2.7/site-packages;export PYTHONPATH' | sudo tee -a /etc/profile.d/samba4-env-build.sh
     $ . /etc/profile.d/samba4-env-build.sh
 
+# Setting up OpenChange dependencies #
```

Comment:
I am installing Jessie and comparing the packets from https://packages.debian.org/jessie
and http://packages.ubuntu.com. I will update the packets referring to this.